pkg/controller/volume/persistentvolume/pv_controller_test.go

```diff
@@ -478,6 +476,7 @@ func TestAnnealMigrationAnnotations(t *testing.T) {
 	const gceDriver = "pd.csi.storage.gke.io"
 	tests := []struct {
 		name                 string
+		inTreeVolume         bool
```

Comment:
Please add a test with `inTreeVolume: false`

Reply:
Removed this.